php/class-wp-app-manifest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * WP_APP_Manifest class.
+ *
+ * @package PWA
+ */
+
+/**
+ * WP_APP_Manifest class.
+ *
+ * Mainly copied from Jetpack_PWA_Manifest and Jetpack_PWA_Helpers.
+ */
+class WP_APP_Manifest {
+
+	/**
+	 * The theme color to use if no dynamic value is present.
+	 *
+	 * @var string
+	 */
+	const FALLBACK_THEME_COLOR = '#fff';
+
+	/**
+	 * The REST API namespace for the manifest request.
+	 *
+	 * @var string
+	 */
+	const REST_NAMESPACE = 'wp/v2';
```

Comment:
I wonder if a different namespace should be used?

Reply:
**Namespace**

Hi @westonruter,

Good question. Maybe our endpoint could be:
`web-app/v1/manifest`

Instead of the current:
`wp/v2/web-app-manifest`

It looks like `oembed` also [registers](https://github.com/WordPress/wordpress-develop/blob/a830dbcab3147e67b3076d3d15c89975df675fbb/src/wp-includes/class-wp-oembed-controller.php#L35) a different endpoint from most of the Core ones:

`oembed/1.0/embed`

<img width="733" alt="oembed-routes" src="https://user-images.githubusercontent.com/4063887/41785995-e3b43096-7609-11e8-9ad4-bb3650c3b8bc.png">

Reply:
Or `app/v1/web-manifest` which would better align with the `.webmanifest` file extension.

Reply:
Sure, PR #15 changes the endpoint to `app/v1/web-manifest`.